test/node_entry_point.ts

```diff
@@ -0,0 +1,13 @@
+// Setup tests for Zone without microtask support
+import '../lib/zone';
+import '../lib/node/node';
+import '../lib/zone-spec/long-stack-trace';
+
+// Setup test environment
+import './test-env-setup';
+
+// List all tests here:
+import './common/microtasks.spec';
```

Comment:
can we share this list with we the browser somehow? That way a test file added to the common would be added to both together?